cmd/kpod/main.go

```diff
@@ -15,6 +15,7 @@ func main() {
 
 	app.Commands = []cli.Command{
 		launchCommand,
+		tagCommand,
```

Comment:
remove `cmd/kpod/kpod` binary file https://github.com/kubernetes-incubator/cri-o/pull/633/files#diff-4a1de25dca58436eca671d80fca8de65 (not related to this)